js/ui/configuration-popup.js

```diff
@@ -128,6 +132,17 @@ export const ConfigurationPopup = {
               </div>
               <interaction-array-option v-for="(action, index) in uiInteractArr" v-bind:action="action" v-on:input="uiInteractArr[index]=$event" v-bind:key="action[2]"></interaction-array-option>
             </div>
+            <h5 class="modal-title">Position Legende</h5>
+            <div class="config_radio_legend">
+              <div>
+                <input type="radio" id="legend_bottom" name="legendDisplay" value="bottom" v-on:change="changeLegendPosition" v-model="legendPosition">
```

Comment:
Try if we can only use the model and add an observer to trigger `resetCanvas()` when the store value changes.